gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -468,12 +470,29 @@ def run_pca(
     return run_pca_with_relateds(qc_mt, samples_to_drop, n_pcs=n_pcs)
 
 
+def annotate_subpop_meta(ht: hl.Table) -> hl.Table:
+    meta_ht = meta.ht()
+    ht = ht.annotate(**meta_ht[ht.key])
+    ht = ht.annotate(
+        subpop_description=ht.project_meta.subpop_description,
```

Comment:
I know I did this, but looking over again, all we really need is `subpop_description` for this script right? So maybe we just do keep that and then for the plotting you might want to add a little more info if needed? Open to other thoughts though